### DISCUSSION TITLE
can I use this in LazyColumn?

### DISCUSSION THREAD

**Discussion by ShankarKakumani** [PatilShreyas/Capturable#29] [Q&A] [answered]
can I use this in LazyColumn?

**Comment by PatilShreyas** [comment, marked as answer]
Yes, it can be used

**Reply by ShankarKakumani** [reply]
Just achieved it,
but How do I share it using intent.

I am getting ImageBitmap from capturable and I need bitmap to work with. ( saving to cache and creating uri to share )

**Reply by ShankarKakumani** [reply]
not an issue,
just seen the extension,

.asAndroidBitmap()

**Reply by eranio** [reply]
Hi., in my LazyColumn, the onCaptured is called twice (when two items are on the screen) . did any one saw this problem too?

**Comment by enalbornoz** [comment]
Hi! I've some problem using this in a colum with a verticall scroll... all data is loaded but in small screens an scroll appear. So  captureController does not save not visible content.
Any solution for this case?
